sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <Bug category="I18N"/>
```

Comment:
Comment here about how these are permanent, so we understand why and don't re-discuss it.